linode.yml

```diff
@@ -3,10 +3,12 @@
 # Include your site.yml or just use the ceph-ansible sample:
 - include: "{{ lookup('env','CEPH_ANSIBLE') }}/site.yml.sample"
   vars:
-    cluster_network: 192.168.0.0/16
+    monitor_address_block: 192.168.128.0/17
```

Comment:
It really matters that the subnet is correct and consistent.    It might not break every time, but intermittent failures are the really horrible thing to look out for.  In the private IP section of my linode, it gives me "192.168.214.250 / 17", which means that the subnet in CIDR notation is 192.168.128.0/17, not 192.168.0.16.  But I don't know if this is just Newark NJ or everywhere.

Reply:
I think it may be different at the other sites but I'm not sure. Since there shouldn't be a conflict under `192.168.0.0/16` with another network, I don't see the issue? (The only purpose here for `monitor_address_block` is to tell ceph-ansible which IP range the monitor will be listening on and only matters for configuration with ansible.)

Reply:
I guess maybe it works by accident, since linode will not assign an IP address outside the subnet range 192.168.0.0/16.  It would not work if the situation was reversed and the subnet fed to ceph-ansible was smaller than the range used by linode, since then some of the nodes' IP addresses would not be reachable within the Ceph subnets.

Reply:
Right. My understanding is linode the private subnet `192.168.0.0/16` for all datacenters. They may actually be using `192.168.0.0/17` but that doesn't really matter for our purposes with `ceph-ansible`.